src/js/components/DataTable/doc.js

```diff
@@ -327,6 +327,63 @@ export const themeDoc = {
     type: 'object',
     defaultValue: '{}',
   },
+  'dataTable.header.background': {
+    description: 'Any valid Box background value.',
+    type: `string | 
+    { dark: string, light: string } |
+    { 
+      color: { dark: string, light: string } | string, 
+      dark: bool, 
+      image: string, 
+      position: string, 
+      opacity: bool | string, 
+      repeat: no-repeat | repeat, 
+      size: cover | contain | string
+    }`,
+    defaultValue: undefined,
+  },
+  'dataTable.header.border': {
+    description: 'Any valid Box border value.',
+    type: 'string | object',
+    defaultValue: undefined,
+  },
+  'dataTable.header.font.weight': {
```

Comment:
I wonder if this should be `text` instead of `font`. This would allow us to use `color` here as well.

Reply:
That's a good point. I was aiming to mirror other theme areas that use the `font` and weight/size convention, but those might not support color. Let me see if any of those other `font` theme objects support color or not.

Reply:
A follow-up here @ericsoderberghp. Applying `theme.dataTable.header.color` will affect the text color -- as is inline with the convention of other components. the `font` object would purely support weight and size. I can add `header.color` to the docs for clarity here if you are fine with that convention?

Reply:
Thanks for looking into it. I'm OK with what you've already got.